src/protean/core/entity.py

```diff
@@ -64,6 +64,9 @@ def __new__(mcs, name, bases, attrs):
                 klass.meta_.id_field.bind_to_entity(klass, 'id')
                 declared_fields['id'] = klass.meta_.id_field
 
+        # Construct an empty QuerySet associated with this Entity class
+        klass.query = QuerySet(name)
```

Comment:
Do you think we can move this declarartion to the Entity class so that there wont be any pylint errors?